payment-request/payment-request-disallowed-when-hidden.https.html

```diff
@@ -0,0 +1,61 @@
+<!DOCTYPE html>
+<meta charset="utf-8">
+<title>Test for PaymentRequest.show() method - should fail when tab is not visible</title>
+<link rel="help" href="https://w3c.github.io/payment-request/#show-method">
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<script src="/resources/testdriver.js"></script>
+<script src="/resources/testdriver-vendor.js"></script>
+<!-- For minimize() -->
+<script src="/page-visibility/resources/window_state_context.js"></script>
+<script>
+'use strict';
+
+promise_test(async t => {
+  const {minimize, restore} = window_state_context(t);
+
+  const methods = [
+      { supportedMethods: "basic-card" },
+      {
+        supportedMethods: "https://apple.com/apple-pay",
+        data: {
+          version: 3,
+          merchantIdentifier: "merchant.com.example",
+          countryCode: "US",
+          merchantCapabilities: ["supports3DS"],
+          supportedNetworks: ["visa"],
+        },
+      },
+  ];
+
+  const request = new PaymentRequest([
+      { supportedMethods: "basic-card" },
```

Comment:
Is `basic-card` something that WPTs should continue to exercise after it has been deprecated?

Reply:
It is a good question. The Payment Request WPT tests are in a weird place, because there is no longer a 'good' set of methods that work reliably across Chrome and Safari (and any other browser that would implement it).

It's likely that just using just https://apple.com/apple-pay would suffice, because for Chrome that would look like a Payment Handler URL and it would at least accept it as a valid payment method. For this new test I just copied the other tests, which is why it still has basic-card.

What do you think - remove basic-card just here for now, or leave it to match other tests and try to clean them all up in the future?

Reply:
If removing `basic-card` works here, let's remove it for this test. Other tests can be taken care of at a later point.

Reply:
Done.

Reply:
Tests still pass as expected - https://wpt.fyi/results/?sha=ee3b94518e&label=pr_head